common/src/main/java/com/google/auto/common/MoreElements.java

```diff
@@ -189,5 +195,24 @@ public static boolean isAnnotationPresent(Element element,
     return Optional.absent();
   }
 
+  /**
+   * Returns a {@link Predicate} that can be used to filter elements by {@link Modifier}.
```

Comment:
I'm wondering if there would be a way of phrasing the doc comment or changing the name of the method, so that it's more immediately obvious that there can be other modifiers too. Maybe just add a sentence like this:

> The predicate returns {@code true} if the input {@link Element} has all of the given {@code modifiers}, perhaps in addition to others.